lightning-graph-sync/src/lib.rs

```diff
@@ -0,0 +1,241 @@
+use core::ops::Deref;
```

Comment:
Can you set the `deny(missing_docs)` and `forbid(unsafe_code)` attrs?

Reply:
Done